docs/understanding_deis/components.rst

```diff
@@ -27,7 +27,9 @@ platform state. Backups and WAL logs are pushed to :ref:`Store`.
 
 Cache
 -----
-The cache is an instance of `Redis`_ used by the registry.
+The cache component is an optional instance of `Redis`_. If the cache is running,
+it will be used by the :ref:`registry`. The ``deisctl`` provisioning tool does
```

Comment:
We should probably clarify why we disabled it as well, which in this case is because we are running a modified fork of the registry whose endpoints don't play nicely with a cache. However, if the user is not sing `deis pull` then this is a nice optimization.